plugins/woocommerce/client/legacy/js/admin/meta-boxes-product-variation.js

```diff
@@ -1118,25 +1104,15 @@ jQuery( function ( $ ) {
 					woocommerce_admin_meta_boxes_variations.ajax_url,
 					data,
 					function ( response ) {
-						var count = parseInt( response, 10 );
+						let count = parseInt( response, 10 );
+						count = isNaN( count ) ? 0 : count;
```

Comment:
What do you think about doing something like this?
```
const count = parseInt( response, 10 ) || 0;
```

Reply:
For sure... much better. Not sure what I was thinking?! 😄 Done.